docs/proposals/optional_application_loging.md

```diff
@@ -0,0 +1,106 @@
+Optional Application Logging
+============================
+
+The EFK aggregated logging stack by default collects application logs with
+fluentd from STDOUT of each container managed by OpenShift. Currently only way
+of opting out is based on a node label and ansible variable
```

Comment:
We should expand this to clarify:
* opting out the entire node for logging
* explicitly controlled by operator